python/console/console_sci.py

```diff
@@ -82,9 +88,52 @@ def __init__(self):
                 pass
 
     def execCommandImpl(self, cmd):
-        res = self.currentState()
+
+        # Child process running, input should be sent to it
+        if self.currentState() == 2:
```

Comment:
could it make sense to have named constants for the states 0, 1, 2 ?

Reply:
Sure. What names would you suggest? I was thinking 
- PS1 (cf sys.ps1/sys.ps2)
- PS2
- STDIN

Reply:
> Sure. What names would you suggest?

no idea as I'm totally unfamiliar with that piece of code, hence my remark :-)